oap-server/server-starter/src/main/assembly/application.yml

```diff
@@ -78,14 +78,16 @@ receiver-trace:
     slowDBAccessThreshold: ${SW_SLOW_DB_THRESHOLD:default:200,mongodb:100} # The slow database access thresholds. Unit ms.
 receiver-jvm:
   default:
-#service-mesh:
-#  default:
-#    bufferPath: ${SW_SERVICE_MESH_BUFFER_PATH:../mesh-buffer/}  # Path to trace buffer files, suggest to use absolute path
-#    bufferOffsetMaxFileSize: ${SW_SERVICE_MESH_OFFSET_MAX_FILE_SIZE:100} # Unit is MB
-#    bufferDataMaxFileSize: ${SW_SERVICE_MESH_BUFFER_DATA_MAX_FILE_SIZE:500} # Unit is MB
-#    bufferFileCleanWhenRestart: ${SW_SERVICE_MESH_BUFFER_FILE_CLEAN_WHEN_RESTART:false}
-#istio-telemetry:
-#  default:
+service-mesh:
+  default:
+    bufferPath: ${SW_SERVICE_MESH_BUFFER_PATH:../mesh-buffer/}  # Path to trace buffer files, suggest to use absolute path
+    bufferOffsetMaxFileSize: ${SW_SERVICE_MESH_OFFSET_MAX_FILE_SIZE:100} # Unit is MB
+    bufferDataMaxFileSize: ${SW_SERVICE_MESH_BUFFER_DATA_MAX_FILE_SIZE:500} # Unit is MB
+    bufferFileCleanWhenRestart: ${SW_SERVICE_MESH_BUFFER_FILE_CLEAN_WHEN_RESTART:false}
+istio-telemetry:
```

Comment:
Do istio and mesh receivers need to be on by default?

Reply:
I don't know which should be on by default, which should not. This is based on user scenario.

Reply:
I think the simplest configuration to turn on by default is appropriate, turn on it when users need it.

Reply:
Quick start -> Advanced

Reply:
But my confuse is which is the simplest? Mesh, envoy are not additional values for trace receiver, right? So how could be advanced?

Reply:
This is the first one so you confuse which is the necessary one. But some time later, you will confuse that too much receivers are turn on when more and more receivers add in.

Reply:
Yes. I agree on that. 

But which should be off by default is the question. Also, there is one yml included in docker image, which has the same issue. In image, you can't surn on or off dynamicly, unless we load the file from volume.

Anyway, I would say, Yes. Too many receivers is definitely a problem. We need a solution for this, but not just simply keep some of them on or off.

In my mind, if we could turn on receiver by command args or env variables, should be better. Because we could provide more shell scripts to run in different ways. Such as, tracing, mesh, customize modes.

Could we move this to another issue to discuss, and submit pr to solve this?

Reply:
@peng-yongsheng Discussion filed here, https://github.com/apache/incubator-skywalking/issues/2322

Reply:
Agree, keep this PR unchanged, merge it into master.